### PR TITLE
Introduce EnterEmail component

### PR DESCRIPTION
**Pull request by xitronix**
image

### CODE REVIEW

File: universal-login-react/src/ui/Migrating/EnterEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import {PrimaryButton} from '../commons/Buttons/PrimaryButton';
+import '../styles/base/enterEmail.sass';
+import '../styles/themes/UniLogin/enterEmailThemeUniLogin.sass';
+import emailIcon from '../assets/icons/e-mail.svg';
+import {SecondaryButton} from '../commons/Buttons/SecondaryButton';
+import Spinner from '../commons/Spinner';
+import {OnboardingStepsWrapper} from '../Onboarding/OnboardingStepsWrapper';
+import {useHistory} from 'react-router';
+
+const emailValidator = {
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Migrating/EnterEmail.tsx:16]: We can extract it to separate file e.g. `inputValidators` and get rid of duplications

File: universal-login-react/src/ui/Migrating/EnterEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+export const EnterEmail = ({hideModal, walletService, onConfirm}: EnterEmailProps) => {
+  const history = useHistory();
+  const [email, setEmail, emailError] = useInputField([emailValidator]);
+  const [loading, setLoading] = useState(false);
+  const primaryButtonClassName = useClassFor('proceed-btn');
+
+  const onConfirmClick = async () => {
+    ensureNotFalsy(email, Error, 'Email missing');
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Migrating/EnterEmail.tsx:34]: `Email is missing`

File: universal-login-react/src/ui/Migrating/EnterEmail.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    setLoading(true);
+    await onConfirm(email);
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Migrating/EnterEmail.tsx:36]: Is it possible that onConfirm callback will be not provided or won't change the modal state? If so, we could setLoading to false after calling onConfirm

Comment (xitronix) [reply universal-login-react/src/ui/Migrating/EnterEmail.tsx:36]: No, the component doesn't have sens if we are not changing screen

**Review by JustynaBroniszewska — APPROVED** [review]

**Pull request merged**